src/Swarm/Web.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- |
+-- A web service for Swarm.
+--
+-- The service can be started using the `--port 8080` command line argument,
+-- or through the REPL by calling `Swarm.App.demoWeb`.
+--
+-- Once running, here are the available endpoints:
+--
+--   * /robots : return the list of robots
+--   * /robot/ID : return a single robot identified by its id
```

Comment:
Running `curl localhost:8080/robots/1` or even `curl localhost:8080/robots?id=1` does not work for me. :confused:

Reply:
@xsebek the path should be `/robot/1`, without the plural s

Reply:
Oh, I am not a web dev so this feels a bit surprising (also `/robot` silently gives nothing), but I guess it is easier this way.